website/src/pages/help.tsx

```diff
@@ -65,30 +63,27 @@ function Help(props) {
           </div>
           <p>Ask questions about the documentation and project</p>
           <div className="buttons">
-            <a href="https://github.com/apache/apisix/issues" target="_blank">
+            <a href="https://github.com/apache/apisix/issues" target="_blank" rel="noreferrer">
               GitHub
               <ChevronRight />
             </a>
             <a
               href="https://apisix.apache.org/docs/general/join"
               target="_blank"
+              rel="noreferrer"
             >
               Slack
               <ChevronRight />
             </a>
-            <a href="https://twitter.com/ApacheAPISIX" target="_blank">
+            <a href="https://twitter.com/ApacheAPISIX" target="_blank" rel="noreferrer">
```

Comment:
According to https://www.reliablesoft.net/noreferrer-noopener/#noreferrer, I'm not sure if we need to use `noreferrer`?